lib/dictionary_processing.py

```diff
@@ -219,15 +219,6 @@ def check_for_request_mapping_replacement(service, operation_id):
     return False
 
 
-def check_for_rest_navigation_replacement(service_url, rest_navigation_handler):
```

Comment:
I am now concerned, if the list of 6.0 services is not passed, the tool have to figure them out the way it used to do, why is this being deleted?

Reply:
Restored the rest-navigation calls - they are invoked only when the auto-rest-services param is not provided.